config/initializers/training_routes.rb

```diff
@@ -49,3 +49,36 @@
 }.freeze
 
 EARLY_YEARS_ROUTES = TRAINING_ROUTE_AWARD_TYPE.select { |_, v| v == "EYTS" }.keys.freeze
+
+SEED_BURSARIES = [
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:provider_led_postgrad],
+    amount: 24_000,
+    subjects: %w[Chemistry Computing Mathematics Physics],
```

Comment:
Could we name this key "allocation_subjects" so it's clear that it's different to a specialism? In one of our branches (I think John H's) we've put these under constants so would be good to update this when the other is merged in (or vice versa)